python-avocado.spec

```diff
@@ -2,27 +2,27 @@
 
 # Conditional for release vs. snapshot builds. Set to 1 for release build.
 %if ! 0%{?rel_build:1}
-%global rel_build 1
+    %global rel_build 1
 %endif
 
 # Settings used for build from snapshots.
 %if 0%{?rel_build}
-%global gittar		%{srcname}-%{version}.tar.gz
+    %global gittar		%{srcname}-%{version}.tar.gz
 %else
-%if ! 0%{?commit:1}
-%global commit		0baf8a1665941d3e8eec88e56cda414e9d937970
-%endif
-%if ! 0%{?commit_date:1}
-%global commit_date	20170425
-%endif
-%global shortcommit	%(c=%{commit};echo ${c:0:7})
-%global gitrel		.%{commit_date}git%{shortcommit}
-%global gittar		%{srcname}-%{shortcommit}.tar.gz
+    %if ! 0%{?commit:1}
+        %global commit		b45e2c532001187e23cef61212fdc8fe44425a2a
```

Comment:
Don't know why this visual-change patch is changing the hardcoded commit string, but it should not matter anyway (or am I wrong?)